### ISSUE TITLE
Hello hello

### ISSUE THREAD

**Issue by sardonivictus** [MikesMachines/MilkIt#2]
Fantastic little write up, just wanted to say thank you for taking the time to mess with it. Also curious to know if any more interesting progress has been made on the app/code. Been dying to build something like this for ages and I think Im finally going to go for it.

**Comment by MikesMachines** [comment]
Yes, there's major I'm working on. Changing stepper library FastacelStepper. 

The biggest thing I'm working on is using rs485 from the ESP to the motor and use Modbus commands to perform all tasks and leverage the drivers' functionality for homing with no code, and others.

There are reasons updating the step library still need to be done, even with the goal of Modbus to use commands and replace pulse eventually, we need to do step pulses until then. 

I'm really trying to pick a motor to that I'd recommend but it's been frustrating testing. I bought an integrated, closed loop stepper. Runs fine over 485, but doesn't support a pulse step interface
